templates/unattended-upgrades.erb

```diff
@@ -71,6 +71,13 @@ Unattended-Upgrade::MailOnlyOnError "<%= @_mail['only_on_error'].to_s %>";
 Unattended-Upgrade::Sender "<%= @sender %>";
 
 <%- end -%>
+// Remove unused automatically installed kernel-related packages
+// (kernel images, kernel headers and kernel version locked tools).
+Unattended-Upgrade::Remove-Unused-Kernel-Packages "<%= @_auto['remove-unused-kernel'].to_s %>";
```

Comment:
I'm not too familiar with unattended upgrade and older Debian versions. Will this break on older Debian versions that don't support this or is this supported on all Debian versions listed in metadata.json?

Reply:
It will only search for that string in newer versions. I only tested my branch on debian stretch which the supplied packages does not yet support this feature and i didn't had any issues.

https://github.com/mvo5/unattended-upgrades/blob/f758f57a47ab5a38006173e3e6bcb6648f350fb8/unattended-upgrade#L2177

We could of course introduce version depending configs but would probably take some more changes than this simple PR.

Reply:
I think it's not a good pattern to add config options that aren't supported. It probably makes sense to wrap this into an if clause and only add it to the file if the debian version supports it.

Reply:
Perhaps they need to default to undef or not be part of the default. When the value is set, output the setting.